lib/client-assets.php

```diff
@@ -646,7 +646,7 @@ function gutenberg_extend_block_editor_styles( $settings ) {
 	}
 
 	// Remove the default font editor styles for FSE themes.
-	if ( gutenberg_is_fse_theme() ) {
+	if ( gutenberg_is_fse_enabled() ) {
```

Comment:
I feel this should be something more like  `gutenberg_supports_block_templates`. I don't like  us adding "fse" to code.

Reply:
Makes more sense... Naming things is hard 😆 
Changed to ` gutenberg_supports_block_templates` 👍